### DISCUSSION TITLE
is 1vcpu + 1gb ram (60gb ssd) enough for running the message broker ?

### DISCUSSION THREAD

**Discussion by RixkiSSS** [cloudamqp/lavinmq#610] [Q&A] [answered]
i just want to know, since the information not so many described on website, it just like "Low RAM requirements" and or "LavinMQ gives you more bang for the buck by maximizing performance while keeping the footprint on RAM and CPU at a minimum.", well... if using the example on website cloudamqp, which is the most matching spec with 1vcpu + 1gb ram (60gb ssd)...

also want to know, if lavinmq is running on personal vps, the limit that you mention on the pricing page would be no limit right ?

**Comment by viktorerlingsson** [comment, marked as answer]
LavinMQ only needs a few MBs of RAM and disk space to start up, but will use very different amounts of RAM depending on your use case. Without knowing more about your specific use case, it's hard for us to give you any recommendations. LavinMQ will definitely run with those resources though!

There are no limits if you run LavinMQ yourself. The "limits" on the pricing page for CloudAMQP are not really limits, they are more like suggestions on what kind of usage is possible with the assigned resources.
